src/github/controller.py

```diff
@@ -11,6 +11,7 @@
 
 
 def upsert_pull_request(pull_request: PullRequest):
+    logger.info("Updating/creating asana task for pull request")
```

Comment:
If this useful considering we have logs for the if...else case on lines 23 and 28?